### PR TITLE
[PowerBI]

### PR DESCRIPTION
**Pull request by paulculmsee**
---
When submitting a connector, please check the following conditions for your PR.

- [x ] I attest that the connector works and I verified by deploying and testing all the operations.
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee @Laskewitz I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

File: independent-publisher-connectors/PowerBI/apiDefinition.swagger.json
Comment (paulculmsee) [reply independent-publisher-connectors/PowerBI/apiDefinition.swagger.json]: Response schema added

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@paulculmsee We are preparing your connector for the next deployment schedule. Once your connector is onboards the next deployment schedule, it will deploy your connector in our production environments which would typically take between 7-10 business days. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks

**Comment by sriyen-msft** [comment]
@Laskewitz We are not able to create an entry for your connector as your connector folder name is "PowerBI" which is already exists in our repo. Can you please create a new folder called "PowerBI Administration" and move your connector files into it and update your PR. Sorry for the inconvenience.

**Comment by Laskewitz** [comment]
@sriyen-msft - @paulculmsee has committed the changes. It's all ready to go now.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@paulculmsee @Laskewitz We are prepping your connector production deployment; we need to set up an OAuth app and add its client id and secret values to our Production KeyVault. We are unable to set up an OAuth app using our test account as it doesn’t have any administrative rights to the Power BI service.  

Since you are an internal partner, can you please set up the OAuth app and share the client id and secret to us in a secure channel, so that we can upload them to our Prod KV to proceed with production deployment.

@natalie-pienkowska and @mamurshe  - FYI

**Comment by paulculmsee** [comment]
"Since you are an internal partner" <- Not sure what this means/don't think we are an internal partner. @sriyen-msft

**Comment by sriyen-msft** [comment]
> "Since you are an internal partner" <- Not sure what this means/don't think we are an internal partner. @sriyen-msft

@paulculmsee I am assuming that you are from Microsoft and developing an IP connector.

**Comment by sriyen-msft** [comment]
@paulculmsee Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021.

We will begin the deployments on Wednesday, 1st December 2021. If your connector is queued up for Wednesday (1st December 2021) deployment schedule, the production deployment of your connector in all regions will be expected to be completed by Thursday, 9th December 2021. Thanks

**Comment by paulculmsee** [comment]
> @paulculmsee I am assuming that you are from Microsoft and developing an IP connector.

No @Laskewitz and I are external.

**Comment by natalie-pienkowska** [comment]
@sriyen-msft As we discussed, this is one of the 5 connectors that were submitted by Independent Publishers and that should be Certified connectors and standard.

**Comment by natalie-pienkowska** [comment]
Our team is currently working on setting up OAuth for this connector. We will provide updates and comments along the way.

**Comment by natalie-pienkowska** [comment]
Hi @paulculmsee and @Laskewitz this connector is on the deployment train :) It should be deployed by February 7th.

**Comment by Laskewitz** [comment]
Yesssss!! @paulculmsee 🙏

**Comment by natalie-pienkowska** [comment]
Unfortunately, we had some issues with the deployment. @mamurshe is on top of this and will do everything he can to deploy these connectors.

**Comment by sriyen-msft** [comment]
@paulculmsee @Laskewitz Apologies for the delay in response, We are actively working on preparing and deploying your connector to publish environments. As you know that we are deploying the connector as a Standard connector, For that, we need to go thru a few guidelines and approvals from the concerned teams (ex. Power BI API team) which is a bit lengthy process. Please allow us some more time, we will keep you posted with the updates and deployment plan soon. Thank you so much for your patience.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @paulculmsee, I hope you are doing well. I just want to inform you that we are still working on getting internal authorization for your connector. Apologies for taking longer time than usual and if this is causing any inconvenience.
We will keep you posted.
Thank you so much for your patience and understanding.

**Comment by Amjed-Ayoub** [comment]
Hello @paulculmsee,
I hope you are doing well.
I am following up with you on your connector status. Please let us know how you would like to proceed.
Please let us know if you have any questions.
Thank you.

**Comment by paulculmsee** [comment]
> Hello @paulculmsee, I hope you are doing well. I am following up with you on your connector status. Please let us know how you would like to proceed. Please let us know if you have any questions. Thank you.

Hiya @Amjed-Ayoub 

I am back after an extended absence and have been out of the loop. What do you need?

File: independent-publisher-connectors/PowerBI Administration/apiProperties.json
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/PowerBI Administration/apiProperties.json:23]: @paulculmsee may I ask you please to provide the value?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/PowerBI Administration/apiProperties.json
Comment (paulculmsee) [reply independent-publisher-connectors/PowerBI Administration/apiProperties.json]: Have done

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @paulculmsee ,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We will submit for a 1st party app request. After we get the approval, we are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
Hello @paulculmsee, 
I hope you are doing well. 
Apologies for the delay in response, as the connector certification team was in transition, we could not take your connector further.  If you would like to go as independent publisher with your connector. Could you please close this and resubmit as new connector?

**Comment by vmanoharas** [comment]
Hello @paulculmsee, 

As there is no response from your end from more than a month.  I am closing this PR.  Please resubmit with new PR.

**Pull request closed without merging**
